### ISSUE TITLE
enhancement: libmeta-dfg

### ISSUE THREAD

**Issue by datazuul** [MyCoRe-Org/libmeta#7]
Are there any plans to create a library for "xmlns:dv=“http://dfg-viewer.de/“.
(see https://dfg-viewer.de/fileadmin/groups/dfgviewer/METS-Anwendungsprofil_2.3.1.pdf)

If I would contribute this, how to start? Generate from an XSD?

**Comment by rsteph-de** [comment]
I thought about it a while ago, but did not find the time to implement it.
If you would like to participate, I am happy to integrate your contribution.

Suggestions to get started:
- copy one of the smaller modules (like MARC) for maven / directory structure.
- I did not find an XML-Schema for the DFG-Viewer elements so you would have to start with the PDF-documentation and "reverse engineer" the Java model.
(For the other models I took the XML schema files and derived the Java model step by step)
- As module name I would suggest` libmeta-dfgviewer` (`dfg` alone is not very specific)
